bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/system/ITarget.java

```diff
@@ -79,10 +79,6 @@ public static ITarget forTag(Tag tag) {
         return new Some(getRa().getAs(CoordinateParam.Units.HMS));
     }
 
-    public double getRaHours() {
-        return getRa().getAs(CoordinateParam.Units.HMS);
-    }
-
     public Option<String> getRaString(Option<Long> time) {
```

Comment:
In the old model the coordinates are not time-varying. This extended refactoring exercise is simply to introduce the parameter into the API, to make it possible to introduce the new model in place of `ITarget`.